sc-tools/sc-builder/src/sc_builder_runner.cpp

```diff
@@ -80,7 +80,10 @@ try
   if (!params.m_outputPath.empty())
     memoryParams.Insert({"storage", params.m_outputPath});
 
-  ScConfig config{configPath, {"storage", "log_file", "input_path", "input", "output"}, {"extensions"}};
+  ScConfig config{
+      configPath,
+      {"repo_path", "storage", "log_file", "input_path", "input", "output"},
```

Comment:
why aren't log_type and log_level in this set?

Reply:
These options are not options whose values are paths. This list is only for options with paths